### ISSUE TITLE
The multi-root editor example crashes on init

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#6438]
## 📝 Provide a description of requested docs changes

Go to the multi-root editor docs example with changes from the latest master. It crashes on init.

```js
TypeError: Cannot read property 'stylesProcessor' of undefined
    at new u (element.js:114)
    at b.domToView (domconverter.js:430)
    at b.domChildrenToView (domconverter.js:466)
    at domChildrenToView.next (<anonymous>)
    at b.domToView (domconverter.js:445)
    at o.toView (htmldataprocessor.js:79)
    at ke.parse (datacontroller.js:372)
    at Object.callback (datacontroller.js:295)
    at yn._runPendingChanges (model.js:801)
    at yn.enqueueChange (model.js:227)
```

**Comment by oleq** [comment]
It's because https://github.com/ckeditor/ckeditor5/compare/i/6381?expand=1 hasn't been merged  in https://github.com/ckeditor/ckeditor5/issues/6381.

**Comment by mlewand** [comment]
Fixed with 370a378
